pom.xml

```diff
@@ -309,6 +309,10 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-javadoc-plugin</artifactId>
         <version>2.8</version>
+        <configuration>
+          <additionalparam>-Xdoclint:none</additionalparam>
+          <additionalOptions>-Xdoclint:none</additionalOptions>
```

Comment:
What kinds of errors were showing up?

Reply:
Newest JDK breaks build if there is pendencies in source javadoc:

```
------------------------------------------------------------------------
Reactor Summary:

Prevayler .......................................... SUCCESS [  1.008 s]
Prevayler Core ..................................... FAILURE [  2.686 s]
Extras parent ...................................... SKIPPED
Prevayler XStream Serializer ....................... SKIPPED
Spikes parent ...................................... SKIPPED
Prevayler Mirroring ................................ SKIPPED
Prevayler Factory .................................. SKIPPED
Prevayler Log4J Monitor ............................ SKIPPED
Prevayler Tests .................................... SKIPPED
Prevayler Transactions Facade ...................... SKIPPED
Prevayler Memento Framework ........................ SKIPPED
Prevayler Mirroring Tests .......................... SKIPPED
Demos parent ....................................... SKIPPED
Prevayler Demo 1 ................................... SKIPPED
Prevayler JXPath Demo .............................. SKIPPED
Prevayler Scalability Demo ......................... SKIPPED
Prevayler tutorials ................................ SKIPPED
------------------------------------------------------------------------
BUILD FAILURE
------------------------------------------------------------------------
Total time: 3.894 s
Finished at: 2018-11-19T14:47:52-02:00
Final Memory: 15M/249M
------------------------------------------------------------------------
Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:2.8:jar (attach-javadocs) on project prevayler-core: MavenReportException: Error while creating archive: 
Exit code: 1 - /Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/Prevayler.java:23: warning: no @return
  public P prevalentSystem();
           ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/Prevayler.java:28: warning: no @return
  public Clock clock();
               ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/Prevayler.java:34: error: reference not found
   * @see org.prevayler.PrevaylerFactory
          ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/Prevayler.java:36: warning: no @param for transaction
  public void execute(Transaction<? super P> transaction);
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/Prevayler.java:45: warning: no @param for <R>
  public <R> R execute(Query<? super P, R> sensitiveQuery) throws Exception;
               ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/Prevayler.java:45: warning: no @param for sensitiveQuery
  public <R> R execute(Query<? super P, R> sensitiveQuery) throws Exception;
               ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/Prevayler.java:53: error: reference not found
   * @see org.prevayler.PrevaylerFactory
          ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/Prevayler.java:55: warning: no @param for <R>
  public <R> R execute(TransactionWithQuery<? super P, R> transactionWithQuery) throws Exception;
               ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/Prevayler.java:55: warning: no @param for transactionWithQuery
  public <R> R execute(TransactionWithQuery<? super P, R> transactionWithQuery) throws Exception;
               ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/Prevayler.java:58: error: malformed HTML
   * The same as execute(TransactionWithQuery<P,R>) except no Exception is thrown.
                                             ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/Prevayler.java:58: error: bad use of '>'
   * The same as execute(TransactionWithQuery<P,R>) except no Exception is thrown.
                                                 ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/Prevayler.java:62: warning: no @param for <R>
  public <R> R execute(SureTransactionWithQuery<? super P, R> sureTransactionWithQuery);
               ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/Prevayler.java:62: warning: no @param for sureTransactionWithQuery
  public <R> R execute(SureTransactionWithQuery<? super P, R> sureTransactionWithQuery);
               ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/Prevayler.java:16: error: reference not found
 * @see org.prevayler.PrevaylerFactory
        ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/TransactionWithQuery.java:21: error: malformed HTML
   * This method is called by Prevayler.execute(TransactionWithQuery<P,R>) to execute this TransactionWithQuery on the given Prevalent System. See org.prevayler.demos for usage examples.
                                                                    ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/TransactionWithQuery.java:21: error: bad use of '>'
   * This method is called by Prevayler.execute(TransactionWithQuery<P,R>) to execute this TransactionWithQuery on the given Prevalent System. See org.prevayler.demos for usage examples.
                                                                        ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/TransactionWithQuery.java:26: warning: no @return
  public R executeAndQuery(P prevalentSystem, Date executionTime) throws Exception;
           ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/TransactionWithQuery.java:26: warning: no @throws for java.lang.Exception
  public R executeAndQuery(P prevalentSystem, Date executionTime) throws Exception;
           ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/monitor/LoggingMonitor.java:38: warning: no @param for clazz
  protected boolean isInfoEnabled(Class clazz) {
                    ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/monitor/LoggingMonitor.java:38: warning: no @return
  protected boolean isInfoEnabled(Class clazz) {
                    ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/monitor/Monitor.java:18: warning: no @param for clazz
  void notify(Class clazz, String message);
       ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/monitor/Monitor.java:18: warning: no @param for message
  void notify(Class clazz, String message);
       ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/monitor/Monitor.java:23: warning: no @param for clazz
  void notify(Class clazz, String message, Exception ex);
       ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/monitor/Monitor.java:23: warning: no @param for message
  void notify(Class clazz, String message, Exception ex);
       ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/monitor/Monitor.java:23: warning: no @param for ex
  void notify(Class clazz, String message, Exception ex);
       ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/monitor/Monitor.java:28: warning: no @param for clazz
  void notify(Class clazz, String message, File file);
       ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/monitor/Monitor.java:28: warning: no @param for message
  void notify(Class clazz, String message, File file);
       ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/monitor/Monitor.java:28: warning: no @param for file
  void notify(Class clazz, String message, File file);
       ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/monitor/Monitor.java:33: warning: no @param for clazz
  void notify(Class clazz, String message, File file, Exception ex);
       ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/monitor/Monitor.java:33: warning: no @param for message
  void notify(Class clazz, String message, File file, Exception ex);
       ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/monitor/Monitor.java:33: warning: no @param for file
  void notify(Class clazz, String message, File file, Exception ex);
       ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/monitor/Monitor.java:33: warning: no @param for ex
  void notify(Class clazz, String message, File file, Exception ex);
       ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/DeepCopier.java:13: error: exception not thrown: java.lang.ClassNotFoundException
   * @throws ClassNotFoundException
             ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/DeepCopier.java:13: warning: no description for @throws
   * @throws ClassNotFoundException
     ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/DeepCopier.java:14: error: exception not thrown: java.io.IOException
   * @throws IOException
             ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/DeepCopier.java:14: warning: no description for @throws
   * @throws IOException
     ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/DeepCopier.java:16: warning: no @param for original
  public static Object deepCopy(Object original) {
                       ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/DeepCopier.java:16: warning: no @return
  public static Object deepCopy(Object original) {
                       ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/DeepCopier.java:24: warning: no @param for original
  public static Object deepCopy(Object original, Serializer serializer) {
                       ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/DeepCopier.java:24: warning: no @param for serializer
  public static Object deepCopy(Object original, Serializer serializer) {
                       ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/DeepCopier.java:24: warning: no @return
  public static Object deepCopy(Object original, Serializer serializer) {
                       ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/DeepCopier.java:42: warning: no @param for original
  public static Object deepCopyParallel(Object original, Serializer serializer) throws Exception {
                       ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/DeepCopier.java:42: warning: no @param for serializer
  public static Object deepCopyParallel(Object original, Serializer serializer) throws Exception {
                       ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/DeepCopier.java:42: warning: no @return
  public static Object deepCopyParallel(Object original, Serializer serializer) throws Exception {
                       ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/DeepCopier.java:42: warning: no @throws for java.lang.Exception
  public static Object deepCopyParallel(Object original, Serializer serializer) throws Exception {
                       ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/Turn.java:13: warning: no @return
  public static Turn first() {
                     ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/Turn.java:32: warning: no @return
  public synchronized Turn next() {
                           ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/Turn.java:77: warning: no @param for message
  public void abort(String message, Throwable cause) {
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/Turn.java:77: warning: no @param for cause
  public void abort(String message, Throwable cause) {
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/serialization/Serializer.java:23: warning: no @param for stream
  public void writeObject(OutputStream stream, Object object) throws Exception;
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/serialization/Serializer.java:23: warning: no @param for object
  public void writeObject(OutputStream stream, Object object) throws Exception;
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/serialization/Serializer.java:23: warning: no @throws for java.lang.Exception
  public void writeObject(OutputStream stream, Object object) throws Exception;
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/serialization/Serializer.java:30: warning: no @param for stream
  public Object readObject(InputStream stream) throws Exception;
                ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/serialization/Serializer.java:30: warning: no @return
  public Object readObject(InputStream stream) throws Exception;
                ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/serialization/Serializer.java:30: warning: no @throws for java.lang.Exception
  public Object readObject(InputStream stream) throws Exception;
                ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/serialization/DESSerializer.java:42: warning: no @param for delegate
  public DESSerializer(Serializer delegate, byte[] key) throws GeneralSecurityException {
         ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/serialization/DESSerializer.java:42: warning: no @throws for java.security.GeneralSecurityException
  public DESSerializer(Serializer delegate, byte[] key) throws GeneralSecurityException {
         ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/foundation/serialization/Serializer.java:9: error: self-closing element not allowed
 * <p/>
   ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/journal/PersistentJournal.java:41: warning: no description for @param
   * @param directory
     ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/journal/PersistentJournal.java:45: warning: no @param for journalDiskSync
  public PersistentJournal(PrevaylerDirectory directory, long journalSizeThresholdInBytes, long journalAgeThresholdInMillis,
         ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/journal/PersistentJournal.java:45: warning: no @param for journalSuffix
  public PersistentJournal(PrevaylerDirectory directory, long journalSizeThresholdInBytes, long journalAgeThresholdInMillis,
         ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/journal/PersistentJournal.java:45: warning: no @param for monitor
  public PersistentJournal(PrevaylerDirectory directory, long journalSizeThresholdInBytes, long journalAgeThresholdInMillis,
         ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/journal/PersistentJournal.java:45: warning: no @throws for java.io.IOException
  public PersistentJournal(PrevaylerDirectory directory, long journalSizeThresholdInBytes, long journalAgeThresholdInMillis,
         ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/publishing/TransactionPublisher.java:18: warning: no @param for subscriber
  public void subscribe(TransactionSubscriber subscriber, long initialTransaction) throws IOException, ClassNotFoundException;
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/publishing/TransactionPublisher.java:18: warning: no @param for initialTransaction
  public void subscribe(TransactionSubscriber subscriber, long initialTransaction) throws IOException, ClassNotFoundException;
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/publishing/TransactionPublisher.java:18: warning: no @throws for java.io.IOException
  public void subscribe(TransactionSubscriber subscriber, long initialTransaction) throws IOException, ClassNotFoundException;
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/publishing/TransactionPublisher.java:18: warning: no @throws for java.lang.ClassNotFoundException
  public void subscribe(TransactionSubscriber subscriber, long initialTransaction) throws IOException, ClassNotFoundException;
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/publishing/TransactionPublisher.java:23: warning: no @param for subscriber
  public void cancelSubscription(TransactionSubscriber subscriber);
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/publishing/TransactionPublisher.java:28: warning: no @param for capsule
  public void publish(Capsule capsule);
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/publishing/TransactionPublisher.java:33: warning: no @return
  public Clock clock();
               ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/publishing/TransactionPublisher.java:38: warning: no @throws for java.io.IOException
  public void close() throws IOException;
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/Capsule.java:36: warning: no @return
  public byte[] serialized() {
                ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/Capsule.java:43: warning: no @param for journalSerializer
  public Object deserialize(Serializer journalSerializer) {
                ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/Capsule.java:43: warning: no @return
  public Object deserialize(Serializer journalSerializer) {
                ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/Capsule.java:54: warning: no @param for prevalentSystem
  public void executeOn(Object prevalentSystem, Date executionTime, Serializer journalSerializer) {
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/Capsule.java:54: warning: no @param for executionTime
  public void executeOn(Object prevalentSystem, Date executionTime, Serializer journalSerializer) {
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/Capsule.java:54: warning: no @param for journalSerializer
  public void executeOn(Object prevalentSystem, Date executionTime, Serializer journalSerializer) {
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/Capsule.java:71: warning: no @param for transaction
  protected abstract void justExecute(Object transaction, Object prevalentSystem, Date executionTime);
                          ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/Capsule.java:71: warning: no @param for prevalentSystem
  protected abstract void justExecute(Object transaction, Object prevalentSystem, Date executionTime);
                          ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/Capsule.java:71: warning: no @param for executionTime
  protected abstract void justExecute(Object transaction, Object prevalentSystem, Date executionTime);
                          ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/Capsule.java:76: warning: no @return
  public abstract Capsule cleanCopy();
                          ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:47: error: self-closing element not allowed
   * <p/>
     ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:52: warning: no @param for suffix
  public static void checkValidSnapshotSuffix(String suffix) {
                     ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:61: error: self-closing element not allowed
   * <p/>
     ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:66: warning: no @param for suffix
  public static void checkValidJournalSuffix(String suffix) {
                     ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:78: warning: no @param for version
  public File snapshotFile(long version, String suffix) {
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:78: warning: no @param for suffix
  public File snapshotFile(long version, String suffix) {
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:78: warning: no @return
  public File snapshotFile(long version, String suffix) {
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:88: warning: no @param for transaction
  public File journalFile(long transaction, String suffix) {
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:88: warning: no @param for suffix
  public File journalFile(long transaction, String suffix) {
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:88: warning: no @return
  public File journalFile(long transaction, String suffix) {
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:104: error: self-closing element not allowed
   * <p/>
     ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:107: warning: no @param for file
  public static long snapshotVersion(File file) {
                     ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:107: warning: no @return
  public static long snapshotVersion(File file) {
                     ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:113: error: self-closing element not allowed
   * <p/>
     ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:116: warning: no @param for file
  public static long journalVersion(File file) {
                     ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:116: warning: no @return
  public static long journalVersion(File file) {
                     ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:129: error: self-closing element not allowed
   * <p/>
     ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:132: warning: no @return
  public File latestSnapshot() throws IOException {
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:132: warning: no @throws for java.io.IOException
  public File latestSnapshot() throws IOException {
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:151: error: self-closing element not allowed
   * <p/>
     ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:154: warning: no @param for initialTransactionWanted
  public File findInitialJournalFile(long initialTransactionWanted) {
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:154: warning: no @return
  public File findInitialJournalFile(long initialTransactionWanted) {
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:183: warning: no @param for prefix
  public File createTempFile(String prefix, String suffix) throws IOException {
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:183: warning: no @param for suffix
  public File createTempFile(String prefix, String suffix) throws IOException {
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:183: warning: no @return
  public File createTempFile(String prefix, String suffix) throws IOException {
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:183: warning: no @throws for java.io.IOException
  public File createTempFile(String prefix, String suffix) throws IOException {
              ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:190: warning: no @param for journalFile
  public static void renameUnusedFile(File journalFile) {
                     ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:197: error: self-closing element not allowed
   * <p/>
     ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:201: warning: no @return
  public Set necessaryFiles() throws IOException {
             ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerDirectory.java:201: warning: no @throws for java.io.IOException
  public Set necessaryFiles() throws IOException {
             ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerImpl.java:34: warning: no description for @param
   * @param journalSerializer
     ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerImpl.java:36: warning: no @param for transactionDeepCopyMode
  public PrevaylerImpl(GenericSnapshotManager<P> snapshotManager, TransactionPublisher transactionPublisher,
         ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerImpl.java:36: warning: no @throws for java.io.IOException
  public PrevaylerImpl(GenericSnapshotManager<P> snapshotManager, TransactionPublisher transactionPublisher,
         ^
/Users/dyorgio/works/github/dyorgio/prevayler/core/src/main/java/org/prevayler/implementation/PrevaylerImpl.java:36: warning: no @throws for java.lang.ClassNotFoundException
  public PrevaylerImpl(GenericSnapshotManager<P> snapshotManager, TransactionPublisher transactionPublisher,
         ^

Command line was: /Library/Java/JavaVirtualMachines/jdk1.8.0_192.jdk/Contents/Home/bin/javadoc @options @packages

Refer to the generated Javadoc files in '/Users/dyorgio/works/github/dyorgio/prevayler/core/target/apidocs' dir.
```